src/components/tasks/card/taskLevelEdit.component.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useRef } from "react";
+import TaskLevelType from "@/interfaces/taskLevel.type";
+import classNames from '@/components/tasks/card/card.module.scss';
+
+type TaskLevelPropsType = {
+    taskLevel?: TaskLevelType
+    updateTaskLevelItems: (newTaskLevelValue: TaskLevelType) => void
+}
+
+const categories = {
+    options: ["Frontend", "Backend", "System design"] as string[]
+}
+
+const level = {
+    options: [1,2,3,4,5] as number[]
```

Comment:
I believe level and categories options should be fetched from the backend otherwise for every future extension or iteration of this feature we would have to work on the status site again which can be avoided if put these options on the backend which can later be fetched. 
What do you think...

Reply:
yes, will look into this and see how I can implement that

Reply:
moved these values to constants -> task-level.js